Makefile.shared

```diff
@@ -269,8 +268,7 @@ utils/config.ml: utils/config.mlp config/Makefile
 	    -e 's|%%TARGET%%|$(TARGET)|' \
 	    -e 's|%%WITH_FRAME_POINTERS%%|$(WITH_FRAME_POINTERS)|' \
 	    -e 's|%%WITH_PROFINFO%%|$(WITH_PROFINFO)|' \
-	    -e 's|%%WITH_SPACETIME%%|$(WITH_SPACETIME)|' \
-	    utils/config.mlp > utils/config.ml
+	    -e 's|%%WITH_SPACETIME%%|$(WITH_SPACETIME)|' $< > $@
```

Comment:
I would prefer the "$< > $@" to be kept on a separate line (in order to make the addition of the WITH_SUPERDUPER_EVERYTHING option simpler).